README.md

```diff
@@ -6,12 +6,15 @@ Inter UI is a typeface specially designed for user interfaces.
 
 ### [⬇︎ Download the latest release](https://github.com/rsms/inter/releases)
 
+### Windows and Mac Users
```

Comment:
Could you drop this?
(99.9% of people downloading Inter UI will be on Windows or Mac.)